components/Header/style.ts

```diff
@@ -1,6 +1,7 @@
 import styled from '@emotion/styled';
 
 export const HeaderContainer = styled.div`
+  padding: 1.75rem 0 0 0;
```

Comment:
```suggestion
  padding-top: 1.75rem;
```
한 방향만 값을 주는 경우 padding-top으로 사용하면 더 좋을 것 같아요!

Reply:
8e37fc7b097703967ae3824d7720dd3e6636dffb
수정했습니다!